### PR TITLE
Fix SSR issues with Nextjs

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

**Review by adrian-potepa — APPROVED** [review]

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -5797,6 +5808,12 @@ packages:
       '@types/node': 16.11.27
     dev: true
 
+  /@types/lodash-es/4.17.6:
+    resolution: {integrity: sha512-R+zTeVUKDdfoRxpAryaQNRKk3105Rrgx2CFRClIgRGaqDTdjsm8h6IYA8ir584W3ePzkZfst5xIgDwYrlh9HLg==}
+    dependencies:
+      '@types/lodash': 4.14.182
+    dev: false
+
   /@types/lodash/4.14.182:
```
Comment (jwm0) [inline pnpm-lock.yaml:5817]: old lodash still present, looks like imports were not updated to use lodash-es

File: packages/ui-kit/package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "typescript": "^4.5.3"
   },
   "dependencies": {
-    "@headlessui/react": "^1.5.0",
+    "@headlessui/react": "^1.6.0",
     "clsx": "^1.1.1",
     "react-dom": "18.0.0"
```
Comment (jwm0) [inline packages/ui-kit/package.json:51]: make React + React DOM a peer dependency

File: packages/ui-kit/react_import.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import React from 'react'
```
Comment (jwm0) [inline packages/ui-kit/react_import.js:1]: What is this for?

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: packages/ui-kit/react_import.js (hunk 3 above)
Comment (krzysztofwolski) [reply packages/ui-kit/react_import.js:1]: https://github.com/egoist/tsup/issues/390#issuecomment-933488738
I'll add comment to the code for future reviewers

**Review by jwm0 — APPROVED** [review]

**Pull request merged**